pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorGlobalNSTest.java

```diff
@@ -45,21 +45,21 @@ public void beforeMethod(Method m) {
     }
 
     @Override
-    @BeforeClass(timeOut = 60000)
+    @BeforeClass(timeOut = 300000)
```

Comment:
I think we have some default time out configured for each test.?

Reply:
That gets applied to the `@Test` method but not to `@BeforeClass`, `@BeforeMethod`. Not sure if there's a way to set a default timeout there.